### ISSUE TITLE
Differense between temperature steps in main and menu

### ISSUE THREAD

**Issue by pendragon1** [KartoffelToby/better-thermostat-ui-card#130]
I'm very happy with the card, but I noticed something. When changing temperature in the main screen it only changes in 1 degree, while in the popup Menu I can change it in 0.5 degrees. Can this somehow be changed or like in the thermostat-dark-card where you can set the step yourself.

Mucht appreciated!

**Comment by PdKingz** [comment]
If you want to, you can change this in the source code of the plugin. I know it's not ideal, but as a work-around you can change this in the source code of the plugin. Search for `this.step=1` and replace with `this.step=0.5` on line 179 and 414. Remember to remove the .js.gz file, as this is preferred over the .js file.

**Comment by pendragon1** [comment]
> I know it's not ideal, but as a work-around you can change this in the source code of the plugin. Search for `this.step=1` and replace with `this.step=0.5` on line 179 and 414. Remember to remove the .js.gz file, as this is preferred over the .js file.

Thanks for the reply! Unfortunately this didn't work. Still steps in the main screen is 1. I removed the js.gz file.

**Comment by PdKingz** [comment]
Have you restarted your HA and refreshed your browser cache (CTRL + SHIFT + R)?

**Comment by nelissuh** [comment]
I would also love to reduce the std. step from 1 to 0,5 °C

**Comment by pendragon1** [comment]
> Have you restarted your HA and refreshed your browser cache (CTRL + SHIFT + R)?

Yes I did, but no difference. I tried other thermostats, like the mushroom-climate-card and the default thermostat in HA and they both do 0.5 by default. When I use this thermostat (which I prefer) I only have 0.5 when I click on the 3 dots (menu).

**Comment by maximilianovermeyer** [comment]
Linked to #100

**Comment by markstamnl** [comment]
Create a customize.yaml for youre thermostat something like below:
```
`climate.thermostaat:
  friendly_name: Thermostaat
  target_temp_step: 0.5`
```

then add the customize.yaml to configuration.yaml
```
homeassistant:
  customize: !include customize.yaml
```
